doc/feature_detection_base_out_vars.csv

```diff
@@ -0,0 +1,16 @@
+Variable Name,Description,Units,Type
+frame,Frame/time/file number; starts from 0 and increments by 1 to N times. ,n/a,int64
+idx,"Feature number within that frame; starts at 1, increments by 1 to the number of features for each frame, and resets to 1 when the frame increments",n/a,int
+hdim_1,"First horizontal dimension in grid point space (typically, although not always, N/S or y space)",Number of grid points,float
+hdim_2,"Second horizontal dimension in grid point space (typically, although not always, E/W or x space)",Number of grid points,float
+num,Number of grid points that are within the threshold of this feature,Number of grid points,int
+threshold_value,Maximum threshold value reached by the feature,Units of the input feature,int
+feature,Unique number of the feature; starts from 1 and increments by 1 to the number of features,n/a,int
```

Comment:
for the 'feature' variable description, I think it would be more clear if you added at the end of the description "increments by 1 to the number of features **identified in all frames**"